lib/compress/zstd_cwksp.h

```diff
@@ -520,6 +525,20 @@ MEM_STATIC void* ZSTD_cwksp_reserve_object(ZSTD_cwksp* ws, size_t bytes)
 
     return alloc;
 }
+/**
+ * with alignment control
+ * Note : should happen only once, at workspace first initialization
+ */
+MEM_STATIC void* ZSTD_cwksp_reserve_object_aligned(ZSTD_cwksp* ws, size_t byteSize, size_t alignment)
```

Comment:
Why do we need alignment smaller than `64`? It seems it might be simpler to just keep 64-byte alignment on all allocations.

Reply:
I guess it could also be done this way.
As align-8 is sufficient, align-64 felt overkill.

Reply:
Agreed. Even if this is a desirable change (why?), this seems like it could be done in a different PR.

Reply:
The new `TMP_WORKSPACE` requires an 8-bytes alignment.
The original `ZSTD_cwksp_reserve_object()` that was used here was aligned on `sizeof(void*)`.
It was breaking tests on 32-bit systems.